app/controllers/api/v1/rooms_controller.rb

```diff
@@ -4,18 +4,20 @@ module Api
   module V1
     class RoomsController < ApiController
       before_action :find_room,
-                    only: %i[show update recordings recordings_processing
+                    only: %i[show update destroy recordings recordings_processing
```

Comment:
I'm starting to have a hard time reading this,
We can invert the logic flow by using `except` instead of `only` 
```
      before_action :find_room, except: %i[index create]
```

Reply:
I prefer that we manually specify rather than invert and assume for all methods